src/bp-core/admin/bp-core-admin-settings.php

```diff
@@ -1885,6 +1989,76 @@ function bp_admin_setting_group_layout_default_option() {
 
 }
 
+/**
+ * Admin Settings for Settings > Groups > Group Directories > Grid style
+ *
+ * @since BuddyBoss [BBVERSION]
+ */
+function bb_admin_setting_group_grid_style() {
+	?>
+	<div class="bb-grid-style-outer">
+		<?php
+		new BB_Admin_Setting_Fields(
+			array(
+				'type'        => 'radio',
+				'id'          => 'bb-group-grid-style-',
+				'label'       => esc_html__( 'Grid Style', 'buddyboss' ),
+				'disabled'    => true,
+				'opt_wrapper' => true,
+				'value'       => 'left',
+				'options'     => array(
+					'left'     => array(
+						'label' => is_rtl() ? 'Right' : 'Left',
+						'class' => 'option opt-left',
+					),
+					'centered' => array(
+						'label' => 'Centered',
+						'class' => 'option opt-centered',
+					),
+				),
+			)
+		);
+		?>
+	</div>
+	<p class="description"><?php esc_attr__( 'Select the style of the of grid layouts. Group avatars and cover images will only be displayed if they are enabled.', 'buddyboss' ); ?></p>
+	<?php
+}
+
+/**
+ * Allow Platform default group element setting field
+ *
+ * @since BuddyBoss [BBVERSION]
+ *
+ * @param array $args Field options.
+ */
+function bb_admin_setting_group_elements( $args ) {
+
+	echo "<div class='bb-group-elements'>";
+	foreach ( $args['elements'] as $element ) {
```

Comment:
Added pre check with  `if ( ! empty( $args[ 'elements' ] ) `

Reply:
@krunal-buddyboss Please check this.